ios/FluentUI/Avatar/Avatar.swift

```diff
@@ -427,18 +427,21 @@ public struct Avatar: View, TokenizedControlView {
             }
```

Comment:
Hmm I realize this is unrelated to your PR but I think the logic here might be different from how FURN handles primary text/secondary text (I think the equivalent we have in FURN is idForColor and name - if both are provided we don't combine them, we just use idForColor). Not sure what the desired behavior is here (or if they even need to be aligned?), but I can update FURN if needed

Reply:
They need to be aligned, or else the hash wouldn't be the same

Reply:
The issue might be that the API for avatar for FURN vs. FUA are different - I'm not sure if we can make the comparison that idForColor = email, idForColor seems a lot more general

<img width="669" alt="Screenshot 2022-11-01 at 10 50 59 AM" src="https://user-images.githubusercontent.com/78454019/199303150-fdebd85d-89f3-4ed5-9f3b-22c206d14f7c.png">

Reply:
Resolving this for now as this conversation is out of scope for this PR, but not disregarding this topic!